sycl/source/detail/scheduler/scheduler.hpp

```diff
@@ -715,6 +715,38 @@ class Scheduler {
 
   friend class Command;
   friend class DispatchHostTask;
+
+  class StreamBuffers {
```

Comment:
We need at least some info on this class.

Reply:
Added info in the style of this file